presto-kudu/pom.xml

```diff
@@ -181,21 +181,23 @@
     </profiles>
 
     <build>
-        <plugins>
-            <!-- optional dependency of kudu-client needed only for javadoc build -->
-            <plugin>
-                <groupId>org.apache.maven.plugins</groupId>
-                <artifactId>maven-javadoc-plugin</artifactId>
-                <configuration>
-                    <additionalDependencies>
-                        <additionalDependency>
-                            <groupId>org.apache.yetus</groupId>
-                            <artifactId>audience-annotations</artifactId>
-                            <version>0.8.0</version>
-                        </additionalDependency>
-                    </additionalDependencies>
-                </configuration>
-            </plugin>
-        </plugins>
+        <pluginManagement>
+            <plugins>
+                <!-- optional dependency of kudu-client needed only for javadoc build -->
+                <plugin>
+                    <groupId>org.apache.maven.plugins</groupId>
+                    <artifactId>maven-javadoc-plugin</artifactId>
+                    <configuration>
+                        <additionalDependencies>
+                            <additionalDependency>
+                                <groupId>org.apache.yetus</groupId>
+                                <artifactId>audience-annotations</artifactId>
```

Comment:
wouldn't it be simpler to have `audience-annotations` as `provided` dependency?

Reply:
Simpler, but seems incorrect, since it won't be provided at runtime and is only used for Javadoc.

I haven't actually looked into this Javadoc build issue. My initial thought is that the Javadoc plugin could be handling this automatically by including optional transitive dependencies, though a quick Google search didn't find anything about this idea.